react-native-cli/index.js

```diff
@@ -221,8 +221,15 @@ function checkNodeVersion() {
 
 function checkForVersionArgument() {
   if (process.argv.indexOf('-v') >= 0 || process.argv.indexOf('--version') >= 0) {
-    var pjson = require('./package.json');
-    console.log(pjson.version);
+    var cliJson = require('./package.json');
+    console.log('react-native-cli: ' + cliJson.version);
+    try {
+      var rnJson = require(REACT_NATIVE_PACKAGE_JSON_PATH());
+      console.log('react-native: ' + rnJson.version);
+    }
+    catch (e) {
+      console.log('react-native: n/a - not currently inside a React Native project directory')
```

Comment:
`not currently inside a React Native project directory` -> `not inside a React Native project directory`